scene/resources/particles_material.cpp

```diff
@@ -312,7 +312,17 @@ void ParticlesMaterial::_update_shader() {
 	code += "\n";
 
 	if (emission_shape == EMISSION_SHAPE_POINTS || emission_shape == EMISSION_SHAPE_DIRECTED_POINTS) {
-		code += "	int point = min(emission_texture_point_count - 1, int(rand_from_seed(alt_seed) * float(emission_texture_point_count)));\n";
+		switch (emission_points_order) {
+			case EMISSION_POINTS_ORDER_RANDOM:
+				code += "	int point = min(emission_texture_point_count - 1, int(rand_from_seed(alt_seed) * float(emission_texture_point_count)));\n";
+				break;
+			case EMISSION_POINTS_ORDER_NORMAL:
+				code += "   int point = min(emission_texture_point_count -1, int(base_number) % emission_texture_point_count);\n";
```

Comment:
Use a tab character instead of spaces:

```suggestion
				code += "	int point = min(emission_texture_point_count -1, int(base_number) % emission_texture_point_count);\n";
```